include/dlaf/eigensolver/tridiag_solver/merge.h

```diff
@@ -397,17 +272,18 @@ inline SizeType stablePartitionIndexForDeflationArrays(SizeType n, const ColType
 }
 
 template <Device D>
-auto stablePartitionIndexForDeflation(SizeType i_begin, SizeType i_end, Matrix<const ColType, D>& c,
-                                      Matrix<const SizeType, D>& in, Matrix<SizeType, D>& out) {
+auto stablePartitionIndexForDeflation(const SizeType i_begin, const SizeType i_end,
+                                      Matrix<const ColType, D>& c, Matrix<const SizeType, D>& in,
+                                      Matrix<SizeType, D>& out) {
   namespace ex = pika::execution::experimental;
   namespace di = dlaf::internal;
 
-  constexpr auto backend = dlaf::DefaultBackend_v<D>;
+  constexpr auto B = dlaf::DefaultBackend_v<D>;
```

Comment:
This is peak bikeshedding, and I'm happy to take this discussion to a less formal coffee discussion, but shouldn't this stay `constexpr auto backend`? This is a local variable and we've been naming variables with snake case. `B` in `template <Backend B>` is a template parameter and we've been naming them with capital case.

Reply:
you are right... This is the result of a find replace that went off of its scope... 😉